pkg/ui/workspaces/cluster-ui/src/insights/types.ts

```diff
@@ -177,12 +177,10 @@ export const slowExecutionInsight = (
   execType: InsightExecEnum,
   latencyThreshold?: number,
 ): Insight => {
-  let threshold = latencyThreshold + "ms";
-  if (!latencyThreshold) {
-    threshold =
-      "the value of the 'sql.insights.latency_threshold' cluster setting";
-  }
-  const description = `This ${execType} took longer than ${threshold} to execute.`;
+  const description =
+    `An execution of this ${execType} was marked as slow. Slow executions ` +
```

Comment:
I'm still confused about the original ticket: how does it happen that we capture with elapsed time 70ms when setting is at 100ms? Is the implication that the setting was changed in that scenario?

Reply:
Thank you for the detailed explanation 🙏🏼